scripts/cdk-deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Echo environment information
+echo "Running CDK synth with DC_ENVIRONMENT=$DC_ENVIRONMENT"
+
+# If AWS_PROFILE exists (+x) then see if it's set.
+# Useful if calling script locally.
+if [ -n "${AWS_PROFILE+x}" ]; then
+  echo "Using AWS_PROFILE=$AWS_PROFILE"
+fi
+
+# Check if CDK is available in node_modules
+if [ -f "./node_modules/.bin/cdk" ]; then
+  echo "Using CDK from node_modules"
+  uv run ./node_modules/.bin/cdk deploy "$@"
+else
+  echo "Error: CDK not found in node_modules. Make sure it's installed with 'npm ci'"
+  exit 1
+fi
```

Comment:
In these shell scripts that invoke CDK, you could consider using `npx --no cdk`, which will invoke `./node_modules/.bin/cdk` if it exists and spit out an error if not. `--no` tells it not to download it from NPM if it is not installed.

Reply:
Yeah that's better.

Reply:
[36b4710](https://github.com/DemocracyClub/dc-data-baker/pull/23/commits/36b47107b2cae944d55b3a9dc3cd98af021e3c62) & [ed3f7ea](https://github.com/DemocracyClub/dc-data-baker/pull/23/commits/ed3f7ea044c116cd21a4fb486389579b9b1391ba)

'`uv run npx --no cdk {command i actually care about}`' does feel a little cursed, but that's why it's hidden in a script I suppose :D